lib/redis_client.rb

```diff
@@ -600,9 +604,10 @@ def _coerce!(results)
   end
 
   class Pipeline < Multi
-    def initialize(_command_builder)
-      super
+    def initialize(_command_builder, raise_exception: true)
+      super(_command_builder)
       @timeouts = nil
+      @raise_exception = raise_exception
```

Comment:
@casperisfine should these also be renamed to `exception`. I feel like `exception` would be more suitable for an **exception object itself** and not a boolean flag such as `raise_exception`. What do you think?

Reply:
FYI i went ahead and changed it to `exception` as well. Let me know if that ok with you.